ultrasonic/src/main/kotlin/org/moire/ultrasonic/fragment/SelectAlbumFragment.kt

```diff
@@ -735,6 +803,11 @@ class SelectAlbumFragment : Fragment() {
     private fun getAlbumList(albumListType: String, albumListTitle: Int, size: Int, offset: Int) {
 
         showHeader = false
+        showSelectFolderHeader = !isOffline(context) &&
```

Comment:
If the useID3 setting is basically a setting which determines, whether to show your music as it is stored hierachically on the file system, or based on its ID3 tags, then for consistency with the Artist View, where the folder selector is also not shown when ID3 is enabled, it should take the same conditions here, IMHO

Reply:
I'm of two minds about what's "proper behavior" but I think the consistency argument is solid. Currently the Artist view doesn't show the header, so the Album view shouldn't either. Thankfully it's trivial to change back and forth so I changed it.